### PR TITLE
taxonomy: issue 7312/7313/7314/7316

### LINKED ISSUE #7312: taxonomy: ingredients: add Cotija cheese
As described at: https://en.wikipedia.org/wiki/Cotija_cheese

(context: I've been preparing a few small ingredient taxonomy edits in #7311 -- but adding a new ingredient seemed worthwhile to track  as a separate issue, since there may be a little more work required than when editing/adding WikiData metadata for an existing entry)

### LINKED ISSUE #7313: taxonomy: ingredients: add Havarti cheese
As described at https://en.wikipedia.org/wiki/Havarti

(see also: #7312; I'm planning to work on this task soon)

### LINKED ISSUE #7314: taxonomy: ingredients: add Colby cheese
As described at https://en.wikipedia.org/wiki/Colby_cheese

(see also: #7312; I'm planning to work on this task soon)

### LINKED ISSUE #7316: taxonomy: ingredients: add Muenster cheese
As described at https://en.wikipedia.org/wiki/Muenster_cheese

:warning: Note that this is _not_ the same as Munster cheese! (which already exists within the `ingredient` taxonomy)

Muenster cheese is from the U.S. and _named after_ a town in the Alsace region; [Munster cheese](https://en.wikipedia.org/wiki/Munster_cheese) is a separate product _made in the vicinity of_ the Alsace region.

### PR DESCRIPTION
**Pull request by benbenben2**
Adding some cheese in ingredients.txt and categories.txt taxonomies.

These are related to issues created by @jayaddison a couple of month ago. I hope that it is fine to work on it.

Resolve #7312 
Resolve #7313
Resolve #7314
Resolve #7316

1-#7313
Because a Havarti cheese is composed of couple of ingredients (milk, salt, etc.), Havarti cheese could be also added in categories.txt taxonomy.
For ingredients.txt,
we added Havarti in ingredients.txt
For categories.txt,
there is already an entry for Havarti cheese: https://world.openfoodfacts.org/category/havarti.
We added more languages for this category: https://www.wikidata.org/wiki/Q1591487

**Remark:** there are some product with Havarti as an ingredient: https://world.openfoodfacts.org/ingredient/Havarti

2- #7312
Because a Cotija cheese is composed of couple of ingredients (milk, salt, etc.), Cotija cheese could be also added in categories.txt taxonomy.
Additional information: https://www.wikidata.org/wiki/Q2718594

For ingredients.txt,
we added Cotija in ingredients.txt
For categories.txt,
we added Cotija in categories.txt

**Remark:** there are some product with Muenster as an ingredient: https://world.openfoodfacts.org/ingredient/cotija

3- #7316
Because a Muenster cheese is composed of couple of ingredients (milk, salt, etc.), Muenster cheese could be also added in categories.txt taxonomy.

For ingredients.txt,
we added languages to Munster in ingredients.txt: https://www.wikidata.org/wiki/Q765540
we added Muenster in ingredients.txt: https://www.wikidata.org/wiki/Q6931973
For categories.txt,
we added Munster in categories.txt
we added Muenster in categories.txt

**Remark:** there are some product with Muenster as an ingredient: https://world.openfoodfacts.org/ingredient/muenster

4- #7314
Because a Colby cheese is composed of couple of ingredients (milk, salt, etc.), Colby cheese could be also added in categories.txt taxonomy.

For ingredients.txt,
we added Colby in ingredients.txt
For categories.txt,
we added Colby in categories.txt

**Remark:** there are some product with Muenster as an ingredient: https://world.openfoodfacts.org/ingredient/Colby

Finally, thank you @jayaddison  to have added links and comments in your issues, that was very helpful.

### CODE REVIEW

**Comment by jayaddison** [comment]
Thank you @benbenben2!  I completely lost track of those issues after planning to work on them; I'm glad the links were helpful.

It's great that you've kept the distinction between Munster / Muenster cheese as well.  No doubt some puzzlement over that in future, but it's real-world detail and I think that's important.

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -39374,6 +39374,50 @@ fr:Fromages d’abbaye
 nl:Abdijkazen, Abdijkaas
 wikidata:en:Q2069008
 
+<en:Cheeses
+en:American cheese
```
[the remark this answers is not in the document]
Comment (benbenben2) [reply taxonomies/categories.txt:39378]: This has been updated for the languages I know and removed for the others.

[the remark this answers is not in the document]
Comment (stephanegigandet) [reply taxonomies/categories.txt:39378]: Perfect, thank you!

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**